packages/manager/apps/hycu/src/pages/order/Order.page.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseLayout,
+  IntervalUnitType,
+  OvhSubsidiary,
+  Price,
+} from '@ovh-ux/manager-react-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { OsdsButton, OsdsText, OsdsTile } from '@ovhcloud/ods-components/react';
+
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import Breadcrumb from '@/components/Breadcrumb/Breadcrumb.component';
+import { useOrderCatalogHYCU } from '@/hooks/order/useOrderCatalogHYCU';
+
+export default function Order() {
+  const { t } = useTranslation('order');
+
+  const { environment } = useContext(ShellContext);
+  const subsidiary = environment.getUser().ovhSubsidiary;
+  const { data: orderCatalogHYCU, isLoading } = useOrderCatalogHYCU(subsidiary);
+
+  const header = {
+    title: t('hycu_order_title'),
+  };
+  const description: string = t('hycu_order_description');
+  const subtitle: string = t('hycu_order_subtitle');
+  const subDescription: string = t('hycu_order_subtitle_description');
+
+  return (
+    <BaseLayout
+      breadcrumb={<Breadcrumb />}
+      header={header}
+      description={description}
+      subtitle={subtitle}
+      subDescription={subDescription}
+    >
+      {orderCatalogHYCU && !isLoading && (
+        <div className="grid gap-8 xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4 mb-8">
+          {orderCatalogHYCU.plans.map((p) => (
```

Comment:
```suggestion
          {orderCatalogHYCU.plans.map((product) => (
```

Personnaly I prefer verbose variable. I don't known the team preference